CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+X.Y.Z Release notes (YYYY-MM-DD)
+=============================================================
+### Enhancements
+* Realm JS now supports all Node.js versions above Node 10.
+* Helper methods 'myinstance.keys()', 'myinstance.entries()' are now available to return the properties of that instance including the realm properties from the prototype.
+* An instance method 'toJSON()' is implemented on every Realm Object class to support 'JSON.stringify()' to work as expected on every instance
+
+### Breaking changes
+* Node.js 8 is not supported anymore.
+* Realm objects properties are now defined as accessors on the instance prototype. (Previously they were defined as values on the instance)
+* Due to accessor properties change above calling 'Object.keys()', 'Object.entries()', 'Object.getOwnPropertyDescriptors()' on a Realm.Object instance (objects from a Realm) will not return the realm schema properties.
+
+### Fixed
+* ECMAScript 2015 Class syntax is fully supported now by moving all properties to be accessors on the instance prototype allowing Realm JS to invoke class constructors as constructors (using 'new' instead of previously calling them as functions with 'call')
+
+### Compatibility
+* Realm Object Server: 3.23.1 or later.
+* APIs are backwards compatible with all previous release of realm in the 2.x.y series.
```

Comment:
why do you think so. This PR does not change any API Realm JS provides. Or maybe I am missing something?

Reply:
It depends - if `Object.keys(obj)` was considered a part of the API, I suppose it does. Although you could argue that was probably never mentioned explicitly in the documentation.

Reply:
Are we really compatible with 2.x.y? 
@kneth I think that all release notes are wrong in this section since we introduced API breaking changes in 3.0? Or am I missing something?

Reply:
Imo Object.keys is not part of the Realm JS API. We have changed the behavior at runtime yes. but this section is for API Realm JS provides, imo. I don't have strong opinion here.

Reply:
Fair enough 👍

Reply:
@bmunkholm I think you - in principle - are right. But back then we agreed not to change it since v3.0.0 was a minor breaking change and only a small number of users would be affected. I suggest that we leave it as it is.

Reply:
@bmunkholm Should we resolve this comment